app/src/main/java/com/gathering/android/event/home/FilterDialogFragment.kt

```diff
@@ -0,0 +1,55 @@
+package com.gathering.android.event.home
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.FragmentManager
+import com.gathering.android.databinding.FrgFilterDialogBinding
+
+
+class FilterDialogFragment(private val eventListViewModel: EventListViewModel) : DialogFragment() {
+
+    private lateinit var binding: FrgFilterDialogBinding
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+
+        binding = FrgFilterDialogBinding.inflate(layoutInflater)
+
+        val builder =
+            AlertDialog.Builder(requireContext(), com.gathering.android.R.style.MyAlertDialogStyle)
+        builder.setView(binding.root)
+
+        val dialog = builder.create()
+        dialog.setCanceledOnTouchOutside(false)
+
+        binding.btnFilterCancel.setOnClickListener {
+            dialog.dismiss()
+        }
+        binding.btnFilterApply.setOnClickListener {
+            eventListViewModel.onFilterChanged(
+                Filter(
+                    binding.myContactEvents.isChecked,
+                    binding.myEvents.isChecked,
+                    binding.todayEvents.isChecked
+                )
+            )
+            dialog.dismiss()
+        }
+        return dialog
+    }
+
+    companion object {
+        const val TAG = "filter_dialog"
+    }
+
+    fun showFilterDialog(fragmentManager: FragmentManager) {
```

Comment:
you dont need a new method for this. call the show method from where you called it before. instead make the TAG public and use it there.